hadoop-ozone/objectstore-service/src/main/java/org/apache/hadoop/ozone/web/storage/DistributedStorageHandler.java

```diff
@@ -478,6 +482,28 @@ public OutputStream newKeyWriter(KeyArgs args) throws IOException,
     return new OzoneOutputStream(keyOutputStream);
   }
 
+  /**
+   * Helper function to get default acl list for current user.
+   *
+   * @return listOfAcls
+   * */
+  private List<OzoneAcl> getAclList(KeyArgs args) {
```

Comment:
Same as comment earlier. This should be a common util helper to avoid duplicated code.

Reply:
Here function argument is different. Rpc client uses ugi while all info is derived from KeyArgs.